packages/sdk/src/marketplaces/Opensea.ts

```diff
@@ -221,6 +231,43 @@ function toUnitAmount(amount: bigint, decimals?: number): number {
     : Number(amount);
 }
 
+function normalizeAssets(assets: OpenSeaAsset[]): NormalizedAsset[] {
+  return assets.map((asset) => {
+    return {
+      contractAddress: asset.assetContract.address,
+      tokenId: asset.tokenId!,
+      type: asset.assetContract.schemaName,
+      amount: "1",
+    };
+  });
+}
+
+// Opensea OrderV2 contains price on root level, but payment token info in either maker or taker bundle
+// for now we are working only with orders that use single erc20 payment token
+function normalizeOrder(order: OpenseaOriginalOrder): OpenseaOrderData {
+  const isSellOrder = order.side === ORDER_SIDE_SELL;
+  const erc20Asset = {
+    contractAddress: (isSellOrder
+      ? order.takerAssetBundle
+      : order.makerAssetBundle
+    ).assets[0].assetContract.address,
+    type: "ERC20",
+    amount: new BigNumber(order.currentPrice).toString(),
```

Comment:
Curious question - isn't `order.currentPrice` already a string? Why is this double conversion required?